Config.h

```diff
@@ -0,0 +1,134 @@
+#ifndef _config_
+#define _config_
+
+#include <algorithm>
+
+namespace Config{  
+  // math general --> from namespace TMath
+  static constexpr const float    PI    = 3.14159265358979323846;
```

Comment:
static (internal linkage) is implied for const objects in a namespace -- so drop static.

constexpr implies const so you can use constexpr as the only specifier.